TASVideos/Models/TrimStringModelBinder.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Metadata;
+
+namespace TASVideos.Models;
+
+public class TrimStringModelBinder : IModelBinder
+{
+	public Task BindModelAsync(ModelBindingContext bindingContext)
+	{
+		string? value = bindingContext.ValueProvider.GetValue(bindingContext.ModelName).FirstValue;
+		if (!string.IsNullOrWhiteSpace(value))
```

Comment:
This will allow strings with `Length > 0` that are entirely whitespace through.  Is that intended?

Reply:
Yes, whitespace strings are already handled in default Asp Net Core stuff. They have weird behavior so I changed the code to only Trim actual content. See how it was before f2edc39b658df491d5d2f5996a316c8a506bbe28 .